### ISSUE TITLE
Add postal code to DBS-derived entries

### ISSUE THREAD

**Issue by acka47** [hbz/lobid-organisations#50]
All [DBS-derived entries](http://beta.lobid.org/organisations/search?q=@id=DBS) are currently lacking postal code information. We should add this. This is probably best done based on the "Regionalschlüssel".

**Comment by philboeselager** [comment]
As I understand correctly, a postal code (PLZ) can not always be predicted from the Regionalschlüssel (RS), see https://fragdenstaat.de/anfrage/gemeindeverzeichnis/ , saying: "Da eine Gemeinde, z.B. Großstadt mehrere Postleitzahlen führen kann, werden Sie daher nicht alle PLZ finden können."
@acka47 Do you agree then to infer PLZ from the address information (town and street+nr) ?

**Comment by philboeselager** [comment]
@acka47 : ping ping?

**Comment by acka47** [comment]
> @acka47 Do you agree then to infer PLZ from the address information (town and street+nr) ?

+1

**Comment by philboeselager** [comment]
ES Index on q.....1, Index: "organisations-20151111-pvb"

**Comment by acka47** [comment]
Something went wrong there. Obviously, the postal code was added to field `streetAddress`overwriting the existing value. [Example](http://test.lobid.org/organisations/DBS-GJ698#!) (snippet):

```javascript
{
	"id": "http://beta.lobid.org/organisations/DBS-GJ698#!",
	"location": [
		{
			"type": "http://schema.org/Place",
			"address": {
				"streetAddress": "84164",
				"addressLocality": "Moosthenning",
				"addressCountry": "Germany",
				"type": "http://schema.org/PostalAddress"
				}
			}
		]
}
```

One [beta](http://beta.lobid.org/organisations/DBS-GJ698#!) it currently looks like this:

```javascript
{
	"id": "http://beta.lobid.org/organisations/DBS-GJ698#!",
	"location": [
		{
			"type": "http://schema.org/Place",
			"address": {
				"streetAddress": "Lengthaler Str. 19",
				"addressLocality": "Moosthenning",
				"addressCountry": "Germany",
				"type": "http://schema.org/PostalAddress"
				}
			}
		]
}
```

Solution: Add the postal code to the field named `postalCode`.

**Comment by philboeselager** [comment]
Concerning DBS-GJ698: I really tried to re-process this weird result, but it seemed I couldn't... until I realised that the reason is simply in the source file of "active" DBS bibs. (I had used the "complete" DBS data set while trying to re-process at first...) This (new) input file holds "plz" as "plz", and not "street with number". So - how are we going to proceed? I guess, we need a new DBS file.
